src/api/axiosConfig.js

```diff
@@ -1,7 +1,7 @@
 import axios from "axios";
 
 const instance = axios.create({
-  baseURL: "https://ah-codetitans-staging.herokuapp.com/",
+  baseURL: process.env.REACT_APP_API,
```

Comment:
wasn't this changed in a prior PR?

Reply:
Yes, it was. This change happened after rebasing develop.